src/tests/containerizer/cgroups_isolator_tests.cpp

```diff
@@ -1489,7 +1489,7 @@ TEST_F(CgroupsIsolatorTest, ROOT_CGROUPS_CreateRecursively)
   // We should remove cgroups_root after the slave being started
   // because slave will create cgroups_root dir during startup
   // if it's not present.
-  ASSERT_SOME(cgroups::remove(hierarchy.get(), flags.cgroups_root));
+  ASSERT_SOME(os::rmdir(path::join(hierarchy.get(), flags.cgroups_root)));
```

Comment:
Please, specify a second parameter `recursive = false` here and elsewhere:
```c++
os::rmdir(path::join(hierarchy.get(), flags.cgroups_root), false)
```

I'd recommend running tests under `root` before sending a patch:
```shell
cd build
sudo make check
# or
GLOG_v=2 sudo src/mesos-tests --verbose --gtest_repeat=1 --gtest_filter=*
```

Reply:
> Please, specify a second parameter recursive = false here and elsewhere:

Done.

> I'd recommend running tests under root before sending a patch:

Thanks - it's not mentioned in the contribution guide, I'll submit a PR to update it.